packages/dom/src/serialize-cssom.js

```diff
@@ -27,36 +27,45 @@ function createStyleResource(styleSheet) {
 
 export function serializeCSSOM({ dom, clone, resources, cache, warnings }) {
   // in-memory CSSOM into their respective DOM nodes.
-  for (let styleSheet of dom.styleSheets) {
-    if (isCSSOM(styleSheet)) {
-      let styleId = styleSheet.ownerNode.getAttribute('data-percy-element-id');
-      let cloneOwnerNode = clone.querySelector(`[data-percy-element-id="${styleId}"]`);
-      if (styleSheetsMatch(styleSheet, styleSheetFromNode(cloneOwnerNode))) continue;
-      let style = document.createElement('style');
+  let styleSheets = null;
+  // catch error in case styleSheets property is not available (overwritten to throw error)
+  try {
+    styleSheets = dom.styleSheets;
+  } catch {
+    warnings.add('Skipping `styleSheets` as it is not supported.');
+  }
+  if (styleSheets) {
```

Comment:
Instead of adding this `if` and adding indentation to the code we can just add a return after line 35

Reply:
We also have a check for `dom.adoptedStyleSheets` below this, which we can still check for, hence not returning it from here.

Reply:
Oh, my bad, i thought function ends here. For future we should split both into separate functions.